sklearn/inspection/_permutation_importance.py

```diff
@@ -98,6 +98,32 @@ def permutation_importance(estimator, X, y, scoring=None, n_repeats=5,
     ----------
     .. [BRE] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
              2001. https://doi.org/10.1023/A:1010933404324
+
+    Examples
+    --------
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.inspection import permutation_importance
+
+    >>> X = [[1,9,9],[1,9,9],[1,9,9],
+    ...      [0,9,9],[0,9,9],[0,9,9]]
+    >>> y = [1,1,1,0,0,0]
+
+    >>> clf = LogisticRegression().fit(X,y)
+    LogisticRegression(...)
+
+    >>> result = permutation_importance(clf, X, y, n_repeats=10,
+    ... random_state=0)
```

Comment:
Could  you align this line such that `random_state` would be under `clf` of the previous line